src/internal/components/cartesian-chart/scales.ts

```diff
@@ -1,6 +1,7 @@
 // Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
 // SPDX-License-Identifier: Apache-2.0
-import { scaleLinear, scaleLog, scaleTime, scaleBand, ScaleContinuousNumeric, ScaleTime, ScaleBand } from 'd3-scale';
+import { scaleLinear, scaleLog, scaleTime, scaleBand } from '../../vendor/d3-scale';
+import { ScaleContinuousNumeric, ScaleTime, ScaleBand } from 'd3-scale';
```

Comment:
Can we import types from the vendor folder too?

If possible, we should also update our eslint config ([this rule](https://github.com/cloudscape-design/components/blob/main/.eslintrc#L157)) to disallow importing d3-scale anywhere in the codebase.

Reply:
Good point! - I haven't thought about that approach to prevent importing d3 directly! Updated that, it's now part of the PR and d3-scale imports are highlighted by eslint.